src/views/Auth/Login.jsx

```diff
@@ -19,6 +19,11 @@ export default function Login() {
     event.preventDefault();
     const loginWasSuccessful = auth.login(formState.email, formState.password);
 
+    if (loginWasSuccessful) {
+      history.replace(from)
```

Comment:
Careful here with the value of `from`. Note that it's an object that contains the property `pathname`, which has the value you're looking for. In a larger context, you'd need to use object notation in order to access the property from the nested object. Just something to be aware of moving forward! :)